src/components/List.js

```diff
@@ -15,15 +17,26 @@ const List = ({ firestore }) => {
   };
 
   const handleChange = (e, item) => {
+    const daysInterval = secondsToDays(
+      Date.now() / 1000 - item.lastPurchasedDate['seconds'],
+    );
+    const weightedNextPurchaseEstimate = calculateEstimate(
```

Comment:
I'll second what @AliLynne said, great job naming your variables to make it as clear as possible what they represent! 👏👏👏